src/lib/components/cards/cardButton/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { FormEvent, createElement, ReactNode } from 'react';
+
+import styles from './style.module.scss';
+import { arrowRight } from '../icons';
+
+interface Props {
+  title: string;
+  description: string | ReactNode;
+  disabled?: boolean;
+  className?: string;
+}
+
+type ActionProps =
+  | { href: string; onClick?: (e: FormEvent) => void }
+  | { href?: string; onClick: (e: FormEvent) => void };
+
+const CardContent = ({
+  title,
+  description,
+}: {
+  title: string;
+  description: string | ReactNode;
+}) => (
+  <>
+    <div>
+      <div className="p-p--small">{title}</div>
+      <div className="tc-primary-500 p-p">{description}</div>
```

Comment:
I think what @vincentaudoire meant was adding `className="tc-primary-500 p-p"` only when the description prop is a `string`. If the description is a `ReactNode` we should render the prop without the `div` wrapper.

Reply:
Yes, exactly 👍